packages/babel-plugin-transform-bigint/test/fixtures/addition/expected.js

```diff
@@ -0,0 +1,2 @@
+import _BigInt from "bigint-polyfill";
```

Comment:
@wdhorton Sure, go for it. That library may be slower, that's all, but working in browsers wouldn't be bad.